nisystemlink/clients/notebook/models/_query_execution_response.py

```diff
@@ -0,0 +1,78 @@
+from typing import Any, Dict, Optional
+
+from nisystemlink.clients.core._uplink._json_model import JsonModel
+from pydantic import Field
+
+from ._execution import (
+    ExecutionErrorCode,
+    ExecutionPriority,
+    ExecutionResourceProfile,
+    ExecutionStatus,
+    ReportSettings,
+    Source,
+)
+
+
+class QueryExecutionResponse(JsonModel):
```

Comment:
I mentioned in the other PR we should consolidate this response object with the main `Execution` object instead of having separate objects for query vs. create/update. If we do that then the consolidated `Execution` should have your new fields. Otherwise, this object is missing your new fields.